src/System.Diagnostics.TraceSource/src/System/Diagnostics/DefaultTraceListener.cs

```diff
@@ -126,7 +126,7 @@ public override void Write(string message)
 
         /// <devdoc>
         ///    <para>
-        ///       Writes the output followed by a line terminator using <see cref="System.Diagnostics.Debug.Write"/>.
+        ///       Writes the output followed by a line terminator using <see cref="M:System.Diagnostics.Debug.Write(string)"/>.
```

Comment:
❗️ Also incorrect; same as previous

Reply:
Indeed, thank you. I will remove M to stay consistent (as it wasn't tehre before).